submissions/DmytryjK/task_js_oop/js/app.js

```diff
@@ -0,0 +1,158 @@
+"use strict";
+
+const baseConfig = {
+    fieldWidth: 400,
+    fieldHeight: 390, 
+    heroWidth: 100,
+    heroHeight: 170,
+    enemyWidth: 100,
+    enemyLinkImg: 'images/enemy-bug.png',
+    playerLinkImg: 'images/char-cat-girl.png',
+    speedRandom(minSpeed = 150, maxSpeed = 350) {
+        maxSpeed -= minSpeed;
+        return Math.floor(Math.random() * ++maxSpeed) + minSpeed;
+    }
+};
+
+const body = document.querySelector('body'),
+      scoreBlock = document.createElement('div');
+let scoreCount = 0;
+body.prepend(scoreBlock);
+score();
+
+class Enemy {
```

Comment:
The module was dedicated to prototypical syntax. You were expected to master it.
You won't be able to switch to ES6 syntax on a commercial project unless aligned with the project technical leadership.
You do not have to fix this.

Reply:
I'm not sure that I fully understand the essence of the problem. Do you mean that the score block should also have been added to a separate class?

Reply:
I mean ES6 `class` syntax vs pre-ES6 prototype-based syntax. All that outdated `Classname.prototype` staff you may be required to work with on a real-life commercial project.